browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -11021,10 +11021,12 @@ index e37adf8e47a53311e43a24f0d51246588e5882cd..2b1e231bdb8db2631046be1f0fc600a7
 +        m_dragSourceOperationMask = WebCore::anyDragOperation();
 +
 +        if (auto& info = dragItem.promisedAttachmentInfo) {
-+            NSString *utiType = info.contentType;
-+            if (auto attachment = attachmentForIdentifier(info.attachmentIdentifier))
-+                utiType = attachment->utiType();
-+
++            auto attachment = attachmentForIdentifier(info.attachmentIdentifier);
++            if (!attachment) {
++                dragCancelled();
++                return;
++            }
++            NSString *utiType = attachment->utiType();
```

Comment:
I wonder how it compiles given that uriType() returns WTF::String :man_shrugging: